src/index.html

```diff
@@ -12,7 +12,42 @@
       href="styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="sections">
+      <div class="calendar calendar--months-29 calendar--days-sun">
```

Comment:
The classes 'calendar--months-29' and 'calendar--days-sun' do not follow the BEM naming convention. The modifier should describe some property of the block or element, not its state. In this case, it would be better to use something like 'calendar--month-length_29' and 'calendar--start-day_sun'.